desktop-widgets/diveplanner.cpp

```diff
@@ -559,8 +559,8 @@ void PlannerSettingsWidget::settingsChanged()
 		ui.bottomSAC->setSingleStep(0.1);
 		ui.decoStopSAC->setDecimals(2);
 		ui.decoStopSAC->setSingleStep(0.1);
-		ui.bottomSAC->setValue(ml_to_cuft(prefs.bottomsac));
-		ui.decoStopSAC->setValue(ml_to_cuft(prefs.decosac));
+		ui.bottomSAC->setValue(plannerShared::bottomsac());
+		ui.decoStopSAC->setValue(plannerShared::decosac());
```

Comment:
case in point - this code is now the same between the `if` clause and the `else` clause. So having it twice is even more confusing.

Reply:
now this is what I call a good catch, of me sleeping while working !! of course these needs to be moved out after the if/else (that contains other parts).